app/views/condition/edit.erb

```diff
@@ -0,0 +1,61 @@
+<head>
+  <link rel="stylesheet" href = "//maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="/condition/new.css">
+</head>
+
+<div class="transbox">
+  <form class="form-horizontal" action="/stations/<%= @trips.id %>" method="post">
+    <input type="hidden" name="_method" value="PUT" />
+    <fieldset>
+      <legend>Edit Weather Conditions Report</legend>
+      <div class="control-group">
+        <label class="control-label" name ="weather[date]" value="<%= @conditions.date %>"/>Date</label>
+        <div class="controls">
+          <input type="date" placeholder="date">
+        </div>
+      </div>
+      <div class="control-group">
+        <label class="control-label" name ="weather[max_temperature]" value="<%= @conditions.max_temperature %>"/>Temperature High</label>
+        <div class="controls">
+          <input type="integer" placeholder="high">
```

Comment:
move this to input so it updates the database: name ="weather[max_temperature]" value="<%= @conditions.max_temperature %>"/